packages/cli/config/jest.js

```diff
@@ -127,6 +127,7 @@ async function getProjectConfig(targetPath, displayName) {
     collectCoverageFrom: ['**/*.{js,jsx,ts,tsx,mjs,cjs}', '!**/*.d.ts'],
     moduleNameMapper: {
       '\\.(css|less|scss|sss|styl)$': require.resolve('jest-css-modules'),
+      '^d3-(.*)$': 'd3-$1/dist/d3-$1',
```

Comment:
Sorry but this won't work well in practice. If we need to have overrides for individual packages it'll be very hard to maintain (depending on what version of d3 you happen to be on, _transitively_, things may break spectacularly or not, and it'll also depend on what version of the CLI you'd be on at the time).

We'll have to find a way to make this change unnecessary I'm afraid. Not having fully grasped the underlying problem yet, I don't have any great recommendations at this point myself.